scripts/docker_build/sof_builder/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN cd /home/sof && git clone https://github.com/thesofproject/xtensa-overlay.gi
 	mkdir -p /home/sof/work/ && \
 	cd crosstool-ng && git checkout sof-gcc8.1 && \
 	./bootstrap && ./configure --prefix=`pwd` && make && make install && \
-	for arch in byt hsw apl cnl; do \
+	for arch in byt hsw apl cnl imx; do \
 		cp config-${arch}-gcc8.1-gdb8.1 .config && \
```

Comment:
This patch looks good to me. Here it is OK to use arch = imx because both platforms that we will support imx8/imx8m use the same compiler only the integration is different on the SoC

Reply:
It's the same case for cnl & icl / byt & cht / hsw & bdw / apl & kbl & skl etc. They use compiler *named* after first supporting platform. I think it's better to stick to lower amount of symbols. Just go for `imx8` everywhere. Maybe not ideal, but at least consistent

Reply:
@dbaluta @jajanusz 
I will take this not a problem since this comes from ct-ng build guide.

Reply:
@xiulipan  not a problem

Reply:
@dbaluta @xiulipan It's just in scripts/tools so it's a minor issue. I just don't want to have f.e. 3 symbols describing 1 thing in the code. I don't like that we already have platforms like cannonlake and have to pass `cnl` for it to rimage. So please keep it at most 2 symbols - full name and short name.